[2025-04-24T08:05:12.923+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-24T08:05:12.942+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_stock_project.dbt_snapshot manual__2025-04-24T07:41:12.765475+00:00 [queued]>
[2025-04-24T08:05:12.952+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_stock_project.dbt_snapshot manual__2025-04-24T07:41:12.765475+00:00 [queued]>
[2025-04-24T08:05:12.954+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-24T08:05:12.968+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): dbt_snapshot> on 2025-04-24 07:41:12.765475+00:00
[2025-04-24T08:05:12.976+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dbt_stock_project', 'dbt_snapshot', 'manual__2025-04-24T07:41:12.765475+00:00', '--job-id', '379', '--raw', '--subdir', 'DAGS_FOLDER/226_lab2_elt.py', '--cfg-path', '/tmp/tmp_c0qsey0']
[2025-04-24T08:05:12.979+0000] {standard_task_runner.py:105} INFO - Job 379: Subtask dbt_snapshot
[2025-04-24T08:05:12.979+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4343) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-24T08:05:12.980+0000] {standard_task_runner.py:72} INFO - Started process 4344 to run task
[2025-04-24T08:05:13.028+0000] {task_command.py:467} INFO - Running <TaskInstance: dbt_stock_project.dbt_snapshot manual__2025-04-24T07:41:12.765475+00:00 [running]> on host e8b5346887eb
[2025-04-24T08:05:13.113+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt_stock_project' AIRFLOW_CTX_TASK_ID='dbt_snapshot' AIRFLOW_CTX_EXECUTION_DATE='2025-04-24T07:41:12.765475+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-24T07:41:12.765475+00:00'
[2025-04-24T08:05:13.115+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-24T08:05:13.130+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-24T08:05:13.133+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt snapshot --profiles-dir /home/***/.dbt --project-dir /opt/***/dbt']
[2025-04-24T08:05:13.141+0000] {subprocess.py:86} INFO - Output:
[2025-04-24T08:05:15.187+0000] {subprocess.py:93} INFO - [0m08:05:15  Running with dbt=1.8.7
[2025-04-24T08:05:16.359+0000] {subprocess.py:93} INFO - [0m08:05:16  Registered adapter: snowflake=1.8.1
[2025-04-24T08:05:17.127+0000] {subprocess.py:93} INFO - [0m08:05:17  Found 7 models, 10 data tests, 3 sources, 459 macros
[2025-04-24T08:05:17.131+0000] {subprocess.py:93} INFO - [0m08:05:17
[2025-04-24T08:05:17.131+0000] {subprocess.py:93} INFO - [0m08:05:17  Nothing to do. Try checking your model configs and model specification args
[2025-04-24T08:05:17.995+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-24T08:05:18.026+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-24T08:05:18.028+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dbt_stock_project, task_id=dbt_snapshot, run_id=manual__2025-04-24T07:41:12.765475+00:00, execution_date=20250424T074112, start_date=20250424T080512, end_date=20250424T080518
[2025-04-24T08:05:18.068+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-24T08:05:18.082+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-24T08:05:18.084+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
